keyboards/donutcables/budget96/config.h

```diff
@@ -19,9 +19,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
-#define VENDOR_ID       0xDCDC
-#define PRODUCT_ID      0x0002
-#define DEVICE_VER      0x0001
+#define VENDOR_ID       0x4443
```

Comment:
This VID is already in use in VIA. The PIDs do not match, but still.
https://github.com/the-via/keyboards/blob/master/src/other/snagpad/snagpad.json#L3

Reply:
One might say that somebody snagged it first.